scripts/prove-scoped-package-access/main.go

```
@@ -0,0 +1,310 @@
+	if err != nil {
+		return 0, nil, errors.New("invalid request")
+	}
+	if authorization != "" {
+		req.Header.Set("Authorization", authorization)
+	}
+	req.Header.Set("Accept", "application/vnd.oci.image.index.v1+json, application/vnd.oci.image.manifest.v1+json, application/vnd.docker.distribution.manifest.list.v2+json, application/vnd.docker.distribution.manifest.v2+json, application/vnd.github+json")
+	response, err := p.client.Do(req)
+	if err != nil {
+		return 0, nil, errors.New("request unavailable")
+	}
+	defer response.Body.Close()
```

golangci-lint (2.13.2)

[error] Error return value of `response.Body.Close` is not checked

(errcheck)